README.md

```diff
@@ -22,3 +22,5 @@ COSMOSDB_BONUS_DATABASE_NAME
 COSMOSDB_BONUS_KEY
 
 COSMOSDB_BONUS_URI
+INPS_SERVICE_HOST
```

Comment:
Duplicated

Reply:
You need to put a blank line between. Try a preview of the markdown.

Reply:
Fixed in 7bcc678